win32ss/user/ntuser/desktop.c

```diff
@@ -1216,17 +1216,71 @@ IntPaintDesktop(HDC hDC)
         {
             SIZE sz;
             int x, y;
+            int scaledWidth, scaledHeight;
+            int wallpaperX, wallpaperY, wallpaperWidth, wallpaperHeight;
             HDC hWallpaperDC;
 
             sz.cx = WndDesktop->rcWindow.right - WndDesktop->rcWindow.left;
             sz.cy = WndDesktop->rcWindow.bottom - WndDesktop->rcWindow.top;
 
+            if (gspv.WallpaperMode == wmFit ||
+                gspv.WallpaperMode == wmFill)
+            {
+                int scaleNum, scaleDem;
+
+                // if ((sz.cx / gspv.cxWallpaper) > (sz.cy / gspv.cyWallpaper))
```

Comment:
?

Reply:
The comment describe the condition below, which is the condition with division before replace with multiplication.

Reply:
I guess the part I don't understand is why you need to replace the division with multiplication.

Reply:
Because multiplication gives better precision and wallpaper might larger than screen. If I use division it will be 0.

Reply:
Oh, I missed that the cyWallpaper moved to the left and cxWallpaper moved to the right. Yes, that makes sense, thanks.
Maybe the comment could be clarified to make it look less like "this alternative code is sitting around here" and more like an explanation. E.g. `// Precision improvement over (sz.cx / gspv.cxWallpaper > sz.cy / gspv.cyWallpaper)`